File: Easy/twoSum.java
package Easy;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int[] solution = new int[2];
        
        int numLength = nums.length;
        
        for (int i = 0; i < numLength; i++)
        {
            solution[0] = i;
            
            for (int j = i + 1; j < numLength; j++)
            {
                if (nums[i] + nums[j] == target)
                {
                    solution[1] = j;
                    return solution;
                }
            }
        }
        
        return null;
    }
}

File: Easy/twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        if(!(target%2)){
            for(int index = 0; index < nums.size()-1; index++){
                for(int i = index+1; i < nums.size(); i++){
                    if(!(nums[index]%2)){
                        if(!(nums[i]%2)){
                            if(nums[index]+nums[i] == target){
                                return {index, i};
                            }
                        }
                    } else {
                        if(nums[i]%2){
                            if(nums[index]+nums[i] == target){
                                return {index, i};
                            }
                        }
                    }
                }
            }
        } else {
            for(int index = 0; index < nums.size()-1; index++){
                for(int i = index+1; i < nums.size(); i++){
                    if(!(nums[index]%2)){
                        if(nums[i]%2){
                            if(nums[index]+nums[i] == target){
                                return {index, i};
                            }
                        }
                    } else {
                        if(!(nums[i]%2)){
                            if(nums[index]+nums[i] == target){
                                return {index, i};
                            }
                        }
                    }
                }
            }
        }
        return {0, 0};
    }
};